webui/react/src/ioTypes.ts

```diff
@@ -98,12 +98,12 @@ const ioCommandAddress = io.type({
   container_port: io.number,
   host_ip: io.string,
   host_port: io.number,
-  protocol: io.string,
+  protocol: io.union([ io.string, io.undefined ]),
 });
 
 const ioCommandMisc = io.partial({
   experiment_ids: io.union([ io.array(io.number), io.null ]),
-  privateKey: io.string,
+  private_key: io.union([ io.string, io.undefined ]),
```

Comment:
suggestion: do we need this field? since we're going toward using the new API IMO we could only define and use the attributes we need and that should reduce this out of sync type issues

Reply:
yeah wasn't sure if they were needed, but that's a good idea.